storybook/src/docs/components/DataGrid/DataGrid.stories.tsx

```diff
@@ -232,4 +260,114 @@ storiesOf("stories/components/DataGrid", module)
                 <DataGrid rows={exampleRows} columns={columns} />
             </Box>
         );
+    })
+    .add("useDataGridExcelExport", () => {
+        const dataGridProps = useDataGridRemote();
+
+        const variables = {
+            limit: dataGridProps.pageSize,
+            offset: dataGridProps.page * dataGridProps.pageSize,
+            sort: dataGridProps.sortModel[0]?.field,
+            order: dataGridProps.sortModel[0]?.sort,
+        };
+
+        const columns: GridColDef[] = [
+            {
+                field: "mission_name",
+                headerName: "Mission Name",
+            },
+            {
+                field: "launch_date_local",
+                headerName: "Launch Date",
+                type: "dateTime",
+            },
+        ];
+
+        const [showMoreMenu, setShowMoreMenu] = React.useState<boolean>(false);
+        const moreMenuRef = React.useRef<HTMLButtonElement>(null);
+
+        const query = gql`
+            query LaunchesPast($limit: Int, $offset: Int, $sort: String, $order: String) {
+                launchesPastResult(limit: $limit, offset: $offset, sort: $sort, order: $order) {
+                    data {
+                        id
+                        mission_name
+                        launch_date_local
+                    }
+                    result {
+                        totalCount
+                    }
+                }
+            }
+        `;
+
+        const { data, loading, error } = useQuery<GQLQuery, QueryVariables | undefined>(query, {
+            variables,
+        });
+
+        const exportApi = useDataGridExcelExport<Launch, GQLQuery, QueryVariables>({
+            columns,
+            variables,
+            query,
+            resolveQueryNodes: (data) => data.launchesPastResult.data,
+            totalCount: data?.launchesPastResult.result.totalCount ?? 0,
+            exportOptions: {
+                fileName: "ExampleName",
+            },
+        });
+
+        function DemoToolbar() {
+            return (
+                <Toolbar>
+                    <ToolbarFillSpace />
+                    <ToolbarItem>
```

Comment:
This should be `ToolbarActions` as `ToolbarItem` adds a space and a border to the right, which should not be the case for the last item on the right side of the `Toolbar`. 

```suggestion
                    <ToolbarActions>
```